docs/recitations/reci11-p5-checkin.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Recitation 10 - Open Source Projects
+title: Recitation 11 - Open Source Project and Task Selection Check In
 ---
 
 # Recitation 11: Open Source Project and Task Selection Check In
 
 This recitation is dedicated to an **in-person** group presentation about your project and task selection. See [P5A: Project and Task Selection](https://cmu-313.github.io/projects/P5/1_projectcheck/#check-in-presentation-80-pts) for more details.
 
-Make sure to [submit your slides on Gradescope](https://www.gradescope.com/courses/488839/assignments/2826606) before you present.
+Make sure to [submit your slides on Gradescope](https://www.gradescope.com/courses/583198/assignments/3717976) before you present.
```

Comment:
This links to the P5B assignment on gradescope (for presentation slides). Did you mean to link to [P5A](https://www.gradescope.com/courses/583198/assignments/3717965) instead?

Reply:
Thank you for catching this! Just fixed the gradescope link to P5A